resources/files/ruby_219/rbconfig/rbconfig-powerpc64le-linux-gnu.rb

```diff
@@ -196,8 +196,8 @@ module RbConfig
   CONFIG["build_vendor"] = "ibm"
   CONFIG["build_cpu"] = "powerpc64le"
   CONFIG["build"] = "powerpc64le-ibm-linux-gnu"
-  CONFIG["RUBY_RELEASE_DATE"] = "2016-03-30"
-  CONFIG["RUBY_PROGRAM_VERSION"] = "2.3.3"
+  CONFIG["RUBY_RELEASE_DATE"] = "2015-04-13"
+  CONFIG["RUBY_PROGRAM_VERSION"] = "2.1.9"
```

Comment:
Should we have the same` CONFIG["RUBY_PROGRAM_VERSION"]` in `rbconfig-ppc64le-redhat-linux.rb`?